adopt-a-dog/stylesheet.css

```diff
@@ -0,0 +1,226 @@
+  margin-top: 5rem;
+}
+
+.banner {
+  background-image: url("images/christopher-harris-57366-unsplash.jpg");
+  background-position: right;
+  background-repeat: no-repeat;
+  background-size: 85%;
+  height: 40rem;
+  background-color: black;
+  padding-top: 5rem;
+  padding-bottom: 15rem;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+
+}
+
+h1 {
+  color: lightgreen;
+}
+
+h2 {
+  color: steelblue;
+  font-size: 30px;
+  font-weight: bold;
+}
+
+h3, form {
+  color: steelblue;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.dog-profile p {
+  font-size: 18px;
+}
+
+.blog p {
+  font-size: 20px;
+}
+
+main {
+  max-width: 70%;
+  margin: auto;
+}
+
+footer {
+  background-color: steelblue;
+  height: 40rem;
+  padding-top: 5rem;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  color: white;
+}
+
+footer p {
+  display: flex;
+  justify-content: center;
+  padding: 2rem;
+}
+
+header nav {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  background-color: lightgrey;
+}
+
+header nav ul {
+  padding-left: 0;
+  margin-left: 0;
+  max-width: 70%;
+  margin: auto;
+}
+
+header nav li, header nav a {
+  color: steelblue;
+}
+
+header nav a {
+  font-size: 30px;
+}
+
+footer nav li, footer nav a {
+  color: white;
+}
+
+footer nav {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+nav ul {
+  display: flex;
+  justify-content: flex-start;
+
+}
+
+nav li {
+  color: black;
+  font-size: 1rem;
+  font-weight: bold;
+  list-style: none;
+  margin-right: 1rem;
+}
+
+nav li::after {
+  content: '/';
+  margin-left: 1rem;
+}
+
+header nav li::after {
+  font-size: 30px;
+}
+
+nav li:last-child::after {
+  content: ' ';
+  margin-left: 0;
+}
+
+li a {
+  text-decoration: none;
+}
+
+.blog {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.blog div:nth-child(2) {
+  width: 100%;
+  max-width: 50rem;
+  margin: 0 auto 0 auto;
+}
+
+.blog img {
+  width: 100%;
+  max-width: 30rem;
+  height: auto;
+  margin: 0 auto 0 auto;
+}
+
+.dog-profile {
+  width: 20rem;
+  margin: 2rem auto 2rem auto;
+  border-bottom: 1px solid lightgrey;
+  border-left: 1px solid lightgrey;
+  border-right: 1px solid lightgrey;
+}
+
+.dog-profile div:nth-child(2) {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.dog-profile img {
+  width: 20rem;
+  max-height: 10rem;
+  object-fit: cover;
+}
+
+.dog-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.see-more-dogs {
+  background-color: lightgrey;
+  width: 15rem;
+  height: 3rem;
+  color: steelblue;
+  border: none;
+  cursor: pointer;
+  font-size: 25px;
+  font-weight: bold;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin: auto;
+}
+
+.see-more-dogs a {
+  text-decoration: none;
+}
+
+.check-out-form input[type=text], .check-out-form input[type=email] {
```

Comment:
nice!

When making two selectors make sure they're on different lines...for example

h1,
h2 {
  color: red;
}